order/views.py

```diff
@@ -46,26 +39,14 @@ def create(self, request):
                     response = vars(response)
                     response = response["data"]["order"]
                     return Response(response, status.HTTP_201_CREATED)
```

Comment:
hey sorry my bad.. Actually lets not do this.. its not a good idea to automatically create another order..
lets do this:
If you get an invalid signature error, just return resp with 400 bad request, and have the caller call the position id API for themselves and then they'll call the create order API again if they want to create an order.

You'll have to create an endpoint for creating position id. Can do it in this PR itself.

Reply:
but we are going to be the user right or will you make api publicly  available ?